src/openforms/fixtures/cookie_consent.json

```diff
@@ -0,0 +1,88 @@
+[
```

Comment:
Is this file supposed to be loaded for every new installation, or just for test purposes?

If the first, then we should document this or make it part of some hook or playbook?
If the latter, then it should be in `fixtures/test/` imo.

Reply:
It's both - intended as a starting point to have *something* for new installations, and also used in unit tests because it's convenient.

I'll add a data migration that loads the fixture, so it's loaded for new fixtures & can be edited if needed (so we don't overwrite changes made by customers which would happen with a hook or playbook).